### ISSUE TITLE
Represent "user must provide permission" in the schema 

### ISSUE THREAD

**Issue by stephaniehobson** [mdn/browser-compat-data#191]
Features like geolocation, battery, and some types of storage require the user to authorize the site to access that feature. 

We should have a way to record and display when this is the case.

**Comment by Elchi3** [comment]
I think this would be an optional property in `simple_support_statement`:

```
"simple_support_statement": {
  "type": "object",
  "properties": {
    "prefix": { "type": "string" },
    "alternative_name": { "type": "string" },
    "flag": ...,
    "partial_implementation": { "type": "boolean" },
    "user_permission_required: { "type": "boolean" },
    ...
  }
},
```

Does `user_permission_required` sound good to you? Is this a boolean type or something else?

**Comment by stephaniehobson** [comment]
Boolean is good, additional information can be provided in the notes if necessary.

Maybe just `permission_required`? FxOS has some features that the OS only granted permissions to if the developer was trusted. If those cases show up again we could use the same property for those (though, maybe we do not want to do that in which case `user_permission_required` is good.

**Comment by Elchi3** [comment]
See the discussion in https://github.com/mdn/browser-compat-data/issues/190.
tl;dr we are **not** adding a new `user_permission_required property` and sub features should be used if the user permission was required at a later point than the basic support of the API.
